### PR TITLE
move to PackageIcon

### PR DESCRIPTION
**Pull request by SimonCropp**
PackageIconUrl is obsolete

### CODE REVIEW

File: src/Polly/Polly.csproj [hunk 1]
```diff
@@ -15,7 +15,6 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Authors>Michael Wolfenden, App vNext</Authors>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <LangVersion>latest</LangVersion>
```
Comment (martincostello) [inline src/Polly/Polly.csproj:18]: Did you mean to remove this?

File: src/Polly/Polly.csproj [hunk 2]
```diff
@@ -34,15 +33,16 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="..\..\Polly.png" Pack="true" PackagePath="\"/>
```
Comment (martincostello) [inline src/Polly/Polly.csproj:36]:
```suggestion
    <None Include="$(MSBuildThisFileDirectory)..\..\Polly.png" Pack="true" PackagePath="\" />
```

File: src/Polly/Polly.csproj [hunk 3]
```diff
@@ -34,15 +33,16 @@
...
     <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
-      <_Parameter1>Polly.Specs, PublicKey=0024000004800000940000000602000000240000525341310004000001000100150819e3494f97263a3abdd18e5e0c47b04e6c0ede44a6c51d50b545d403ceeb7cbb32d18dbbbcdd1d88a87d7b73206b126be134b0609c36aa3cb31dd2e47e393293102809b8d77f192f3188618a42e651c14ebf05f8f5b76aa91b431642b23497ed82b65d63791cdaa31d4282a2d6cbabc3fe0745b6b6690c417cabf6a1349c</_Parameter1>
+     <_Parameter1>Polly.Specs, PublicKey=0024000004800000940000000602000000240000525341310004000001000100150819e3494f97263a3abdd18e5e0c47b04e6c0ede44a6c51d50b545d403ceeb7cbb32d18dbbbcdd1d88a87d7b73206b126be134b0609c36aa3cb31dd2e47e393293102809b8d77f192f3188618a42e651c14ebf05f8f5b76aa91b431642b23497ed82b65d63791cdaa31d4282a2d6cbabc3fe0745b6b6690c417cabf6a1349c</_Parameter1>
```
Comment (martincostello) [inline src/Polly/Polly.csproj:38]:
```suggestion
      <_Parameter1>Polly.Specs, PublicKey=0024000004800000940000000602000000240000525341310004000001000100150819e3494f97263a3abdd18e5e0c47b04e6c0ede44a6c51d50b545d403ceeb7cbb32d18dbbbcdd1d88a87d7b73206b126be134b0609c36aa3cb31dd2e47e393293102809b8d77f192f3188618a42e651c14ebf05f8f5b76aa91b431642b23497ed82b65d63791cdaa31d4282a2d6cbabc3fe0745b6b6690c417cabf6a1349c</_Parameter1>
```

File: src/Polly/Polly.csproj (hunk 1 above)
Comment (SimonCropp) [reply src/Polly/Polly.csproj:18]: yeah. it is already in the dir props. i can do a new PR if u want

Comment (martincostello) [reply src/Polly/Polly.csproj:18]: No, that's fine. Fewer PRs is less to review 😄

**Review by martincostello — APPROVED** [review]

**Pull request merged**